### PR TITLE
User data is loaded at startup even if he comes from on-boarding

### PR DESCRIPTION
**Pull request by Revenaunt**
When MainActivity is created, user data is loaded every time.

### CODE REVIEW

**Comment by bradmontgomery** [comment]
:+1: , but we should watch this and see if it has any noticeable speed impacts on different devices.

File: src/main/java/org/tndata/android/compass/activity/MainActivity.java [hunk 1]
```diff
@@ -216,7 +216,6 @@ public void onItemClick(int position){
                 break;
 
             case MY_PRIVACY:
-                startActivity(new Intent(getApplicationContext(), PrivacyActivity.class));
```
Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/activity/MainActivity.java]: was this a mistake?

Comment (Revenaunt) [inline src/main/java/org/tndata/android/compass/activity/MainActivity.java]: Woops, I ctrl+z'd more than I should have, my bad.

Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/activity/MainActivity.java]: no worries. That's why it's nice to have a 2nd pair of :eyes:  :)

**Comment by bradmontgomery** [comment]
:+1:

**Pull request merged**